### PR TITLE
Unminimize and add USER envvar

### PR DESCRIPTION
**Pull request by cemiarni**

### CODE REVIEW

File: bldr/data/hooks/pre-install-deps [hunk 1]
```diff
@@ -1 +1,27 @@
...
+
+    if [[ ${unminimize_present} == 0 ]]; then
+        echo
+        cat <<"END"
+Performing an image unminimization, because we experienced that more and more
+packages have tests depending on data which is not presented on a minimized
+image.
+
+If this unminimization bothers you, please remove the
+'perform_unminimization' function from the 'pre-install-debs' hook.
```
Comment (wferi) [inline bldr/data/hooks/pre-install-deps]: pre-install-deps

File: bldr/data/hooks/pre-install-deps [hunk 2]
```diff
@@ -1 +1,27 @@
...
+
+perform_unminimization() {
+    unminimize_cmd=$(command -v unminimize)
+    local unminimize_present=$?
+
+    if [[ ${unminimize_present} == 0 ]]; then
+        echo
+        cat <<"END"
+Performing an image unminimization, because we experienced that more and more
+packages have tests depending on data which is not presented on a minimized
```
Comment (wferi) [inline bldr/data/hooks/pre-install-deps]: present

File: bldr/data/hooks/pre-install-deps [hunk 3]
```diff
@@ -1 +1,27 @@
 #!/bin/bash
+
+perform_unminimization() {
+    unminimize_cmd=$(command -v unminimize)
```
Comment (wferi) [inline bldr/data/hooks/pre-install-deps]: local

File: bldr/data/hooks/pre-install-deps [hunk 4]
```diff
@@ -1 +1,27 @@
...
+    local unminimize_present=$?
+
+    if [[ ${unminimize_present} == 0 ]]; then
```
Comment (wferi) [inline bldr/data/hooks/pre-install-deps]: Or drop this variable and use [[ "${unminimize_cmd}" ]] here.

**Review by wferi — CHANGES_REQUESTED** [review]

File: bldr/data/hooks/pre-install-deps (hunk 3 above)
Comment (cemiarni) [reply bldr/data/hooks/pre-install-deps]: it won't work, because 'local' eats the return code :(

File: bldr/data/hooks/pre-install-deps (hunk 4 above)
Comment (cemiarni) [reply bldr/data/hooks/pre-install-deps]: the current version is more explicit

File: bldr/data/hooks/pre-install-deps (hunk 3 above)
Comment (wferi) [reply bldr/data/hooks/pre-install-deps]: Part of why I recommended dropping `unminimize_present` altogether. But you can make the assignment in a separate command (right after `local`) to work around that.

**Pull request merged**